src/xdocs/beginning_development.xml

```diff
@@ -17,7 +17,7 @@
     </section>
     <section name="Before development">
       <p>
-        1. Ensure that Git, Java JDK >= 11 until JDK 17, maven >= 3.0.1 are installed.<br/>
+        1. Ensure that Git, Java JDK >= 11 until JDK 17, maven >= 3.3.9 are installed.<br/>
         You can find information about development environment preparation here:
```

Comment:
We usually need to update pom.xml and CIs.

Reply:
Updated one entry in the POM. CIs would have been failing if they used a version earlier than this.

Regardless, I couldn't find anything else to update. Let me know if there is anything else.

Reply:
if CI pass, ok to merge